### ISSUE TITLE
"Contrast checker"-plugin: Lack of dark-mode css for contrast ratio number

### ISSUE THREAD

**Issue by eklem** [uxbox/uxbox#10397]
### Steps To Reproduce

1. Set mode to dark-mode
2. Install plugin "Contrast checker"
3. Open plugin "Contrast checker"
4. Select two shapes

### Expected behavior

Ecpected contrast ratio number to be the same color as the label "Contrast ratio:"

### Actual behavior

The contrast ratio number seems to lack a dark-mode color setting, so it's black on dark gray.

### Screenshots or video

<img width="612" height="580" alt="Image" src="https://github.com/user-attachments/assets/09aece5d-054c-46a6-8356-28c60e30d880" />
<img width="643" height="593" alt="Image" src="https://github.com/user-attachments/assets/2f163ed8-1d2c-4b33-8f89-b16cade4ed9e" />

### Desktop (please complete the following information)

- OSX Tahoe 26.5.1 (25F80)
- Firefox 152.0.1 (aarch64)

### Smartphone (please complete the following information)

- not apliccable 

### Environment (please complete the following information)

- Host https://design.penpot.app

### Frontend Stack Trace

```console

```

### Backend Stack Trace

```console

```

### Additional context

Plugin source: https://github.com/penpot/penpot/tree/main/plugins/apps/contrast-plugin

**Comment by Shlok1729** [comment]
Hi! I was looking to pick this issue up, but I tested it on my local environment using the latest develop branch, and it looks like it has already been fixed by a recent update (see screenshot). Does this issue still need to remain open?

<img width="2256" height="1698" alt="Image" src="https://github.com/user-attachments/assets/56193681-5424-442e-bb86-5bb05e07fff6" />

**Comment by eklem** [comment]
Thanks! No, I'll check back in a little while and reopen if still is an issue.
